index.ts

```diff
@@ -68,8 +68,7 @@ async function get<TValue extends Value>(key: string): Promise<TValue | undefine
 
 async function set<TValue extends Value>(key: string, value: TValue, maxAge: TimeDescriptor = {days: 30}): Promise<TValue> {
 	if (typeof value === 'undefined') {
-		// @ts-ignore This never happens in TS because `value` can't be undefined
-		return;
+		await delete_(key);
```

Comment:
I think
```suggestion
		return delete_(key);
```

If not, first you delete it and the. It adds it again later, as undefined.

Reply:
Good point, my bad.